cmd/security-secretstore-setup/res/configuration.toml

```diff
@@ -26,7 +26,7 @@ File = './logs/security-secretstore-setup.log'
 Scheme = "https"
 Server = "edgex-vault"
 Port = 8200
-CertPath = "v1/secret/edgex/edgex-security-proxy-setup/kong-tls"
+CertPath = "/v1/secret/edgex/edgex-security-proxy-setup"
```

Comment:
Investigate. secretstore-setup doesn't use go-mod-secrets and likely writes this location directly.